From 81fd37ec62cf8f064996aac409e6808ec7916153 Mon Sep 17 00:00:00 2001
From: David Edmundson <david@davidedmundson.co.uk>
Date: Sun, 31 Mar 2013 23:18:26 +0100
Subject: [PATCH 3/3] Allow Rich Text in QML notifications plasmoid Allow Rich
 Text in QML Notification plasmoid

This matches behaviour of the notifications in 4.9 which used QLabel
It fixes a problem where we would see notifications with &nbsp; > in the body as we deliberately try and keep formatting the same and be HTML safe.

REVIEW: 109492
---
 .../contents/ui/NotificationDelegate/NotificationDelegate.qml            | 1 +
 1 file changed, 1 insertion(+)

diff --git a/plasma/generic/applets/notifications/contents/ui/NotificationDelegate/NotificationDelegate.qml b/plasma/generic/applets/notifications/contents/ui/NotificationDelegate/NotificationDelegate.qml
index ed8ed99..64d9298 100644
--- a/plasma/generic/applets/notifications/contents/ui/NotificationDelegate/NotificationDelegate.qml
+++ b/plasma/generic/applets/notifications/contents/ui/NotificationDelegate/NotificationDelegate.qml
@@ -203,6 +203,7 @@ PlasmaComponents.ListItem {
                         selectByMouse: true
                         readOnly: true
                         wrapMode: Text.Wrap
+                        textFormat: TextEdit.RichText
                     }
                 }
                 Column {
-- 
1.8.1.4

